### PR TITLE
Ordersummery

### PR DESCRIPTION
**Pull request by mujthabakk**

### CODE REVIEW

File: lib/features/ads/controller/ordercontroller/dateprovider.dart
Comment (thadathilsinan) [inline lib/features/ads/controller/ordercontroller/dateprovider.dart:1]: Rename file

Comment (thadathilsinan) [inline lib/features/ads/controller/ordercontroller/dateprovider.dart:1]: Update logic

File: lib/features/ads/view/widgets/ordersummery/dateselectwidget.dart
Comment (thadathilsinan) [inline lib/features/ads/view/widgets/ordersummery/dateselectwidget.dart:1]: Rename filw

File: lib/features/ads/view/widgets/ordersummery/dateselectwidget.dart [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  final DateTime selectedDate;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final now = selectedDate;
+
+    return Padding(
+      padding: EdgeInsets.only(top: context.spaces.space_125),
+      child: Container(
+        height: 53,
```
Comment (thadathilsinan) [inline lib/features/ads/view/widgets/ordersummery/dateselectwidget.dart:31]: Use spaces

File: lib/main.dart [hunk 2]
```diff
@@ -14,7 +14,33 @@ class MainApp extends ConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     return MaterialApp(
       theme: ref.watch(themeProvider),
-      home: const SplashScreen(),
+      home: const Sapmble(),
     );
   }
 }
+
```
Comment (thadathilsinan) [inline lib/main.dart:21]: Remove this class

**Review by thadathilsinan — CHANGES_REQUESTED** [review]

**Review by Sireen-Official — APPROVED** [review]

**Review by thadathilsinan — APPROVED** [review]

**Pull request merged**